arbeitszeit_flask/translations/de/LC_MESSAGES/messages.po

```diff
@@ -1084,98 +1049,109 @@ msgstr "Verkaufsabweichung"
 msgid "Product description"
 msgstr "Produktbeschreibung"
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:34
-#: arbeitszeit_web/formatters/plan_details_formatter.py:68
-msgid "Planning timeframe (days)"
-msgstr "Planungszeitraum (Tage)"
+#: arbeitszeit_flask/templates/macros/draft_form.html:28
+msgid ""
+"Choose a planning timeframe in which you are planning to produce and deliver "
+"your product."
+msgstr ""
+"Wähle einen Planungszeitraum, in dem du planst, dein Produkt herzustellen und "
+"zu liefern."
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:46
-#: arbeitszeit_web/formatters/plan_details_formatter.py:73
+#: arbeitszeit_flask/templates/macros/draft_form.html:31
+msgid "Planning timeframe"
+msgstr "Planungszeitraum"
+
+#: arbeitszeit_flask/templates/macros/draft_form.html:41
+msgid ""
+"This value is used for informational purposes only, i.e. you do not have to "
+"adhere to any rules here. For example, if you plan to produce beverages, you "
+"are free to choose if your smallest delivery unit is one can or one six-pack "
+"of cans."
+msgstr ""
+"Dieser Wert dient nur zu Informationszwecken, du musst dich hier an keine "
+"Regeln halten. Wenn du z.B. planst, Getränke herzustellen, könnte deine "
+"kleinste Abgabeeinheit sowohl eine Dose als auch ein Sixpack sein."
+
+#: arbeitszeit_flask/templates/macros/draft_form.html:43
+#: arbeitszeit_web/formatters/plan_details_formatter.py:70
 msgid "Smallest delivery unit"
 msgstr "Kleinste Abgabeeinheit"
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:49
+#: arbeitszeit_flask/templates/macros/draft_form.html:46
+msgid "E.g. 1 package, 1 kilogram, 20 bottles, 1 language lesson, etc."
+msgstr "Z.B. 1 Packung, 1 Kilogramm, 20 Flaschen, 1 Sprachstunde, etc."
+
+#: arbeitszeit_flask/templates/macros/draft_form.html:52
+msgid ""
+"How many of your \"smallest delivery unit\" will you produce in total? By "
+"dividing the sum of your production costs by this amount, the app calculates "
+"how much labour time is required to produce one piece of your product."
+msgstr ""
+"Wie viel deiner \"kleinsten Abgabeeinheit\" wirst du insgesamt produzieren? "
+"Indem die App die Summe der Produktionskosten durch diesen Wert teilt, "
+"errechnet sie wie viel Arbeitszeit es braucht, um eine Einheit deines "
+"Produktes herzustellen."
+
+#: arbeitszeit_flask/templates/macros/draft_form.html:64
+#: arbeitszeit_flask/templates/macros/plan_details.html:126
 msgid ""
-"E.g. 1 package of 100 pieces, 1 kilogram, 1 language lesson of 1.5 hour, "
-"etc."
+"This is sometimes also called depreciation, e.g. if you plan to use a machine "
+"that cost 10000 hours to produce, for 10 years, and you have ten one-year-"
+"plans, then for this fixed mean of production, the value would be 1000."
 msgstr ""
-"Z.B. 1 Packung à 100 Stück, 1 Kilogramm, 1 Sprachunterrichtseinheit à 1,5 "
-"Stunden, etc."
+"Manchmal auch Abnutzung genannt, z.B. würdest du bei zehn Ein-Jahres-Plänen, "
```

Comment:
I'd say: Period instead of comma after "genannt". No comma after "Ein-Jahres-Plänen".